docs/content/docs/data-structures/string.md

```diff
@@ -80,6 +79,8 @@ efficient spatial querying of the nearest Marker to a given position. -->
     -   [collaborativeTextArea](https://github.com/microsoft/FluidFramework/blob/main/examples/utils/example-utils/src/reactInputs/CollaborativeTextArea.tsx)
     -   [collaborativeInput](https://github.com/microsoft/FluidFramework/blob/main/examples/utils/example-utils/src/reactInputs/CollaborativeInput.tsx)
 
+<!-- This line ends the content that is copied to the string.md README -->
```

Comment:
We should probably update the embed line numbers to omit this.

Reply:
I'm not able to figure out how this gets here in the first place, the only matches of it I found are in *.md files already (two in docs/ and one in packages/, though).

Reply:
It comes from line 62 of this file. Specifies the line numbers to include from the sequence README

Reply:
Updated!